modules/manage/pages/manage-buckets/migrate-bucket.adoc

```diff
@@ -5,9 +5,9 @@
 
 [.edition]#{enterprise}#
 
-You can migrate a bucket's storage backend if you find the bucket's current backend isn't appropriate. For example, you can migrate a bucket from Couchstore to Magma if the bucket's working set grows beyond its memory quota. 
+You can migrate a bucket's storage backend if you find the bucket's current backend is not appropriate. For example, you can migrate a bucket from Couchstore to Magma if the bucket's working set grows beyond its memory quota. 
```

Comment:
Since we allow migration is both directions - "Couchstore" -> "Magma" and "Magma" -> "Couchstore" we should explicitly mention that somewhere on this page.